py/desitarget/cuts.py

```diff
@@ -2351,7 +2400,8 @@ def apply_cuts(objects, qso_selection='randomforest', gaiamatch=False,
         gaiagmag, gaiabmag, gaiarmag, gaiaaen, gaiadupsource,
         gaiaparamssolved, gaiabprpfactor, gaiasigma5dmax, galb,
         tcnames, qso_optical_cuts, qso_selection,
-        maskbits, Grr, refcat, primary, resolvetargs=resolvetargs
+        maskbits, Grr, refcat, primary, resolvetargs=resolvetargs,
```

Comment:
A little higher up in the code, you'll see:
```
if survey == 'main':
        import desitarget.cuts as targcuts
    elif survey[:2] == 'sv':
        targcuts = import_module("desitarget.{}.{}_cuts".format(survey, survey))
```

This means that if we're running the Main Survey cuts, we import the `set_target_bits()` function from `desitarget.cuts` but if we're running the _SV_ cuts, we import the `set_target_bits()` function from `desitarget.sv1.sv1_cuts`.

So, the _inputs_ to `desitarget.sv1.sv1_cuts.set_target_bits` have to be consistent with the inputs to `desitarget.cuts.set_target_bits`. This is what is causing some of the tests to fail.

Please update `desitarget.sv1.sv1_cuts.set_target_bits()` to have `ra, dec` inputs structured in the exact same fashion as `desitarget.cuts.set_target_bits()`. It's not necessary for `desitarget.sv1.sv1_cuts.set_target_bits()` to _use_ the `ra, dec` inputs, but it will need to _have_ them.

Reply:
OK, i didn't check that.